### ISSUE TITLE
IfcDoorPanelProperties

### ISSUE THREAD

**Issue by lfcastel** [IfcOpenShell/IfcOpenShell#2438]
I'm trying to adapt the DoorPanelProperties of a doortype:
![image](https://user-images.githubusercontent.com/73949503/191950462-166d117e-d660-4524-891c-c1e0abeba149.png)
As you can see in the image, this doortype refers to an instance of IfcDoorPanelProperties through the attribute HasPropertySets where the PanelOperation and PanelPosition attribute are both NOTDEFINED. Verified in notepad (attribute 5 and 8):
#157039=IFCDOORPANELPROPERTIES('1r13uTFJX1Ig3$6x90bRbl',#1,'IfcDoorPanelProperties',$,$,.NOTDEFINED.,$,.NOTDEFINED.,$);

I click on the edit button, but you can't seem to adapt this.

Is this not the right place to edit this? When i go to an object that is typed by this doortype I see the IfcDoorPanelProperties appearing there as a inhereted property set (i'd guess inhereted from the doortype?). When i click edit i get this error:
![image](https://user-images.githubusercontent.com/73949503/191951471-c2cf6557-2a40-4653-b90d-8996bd1435eb.png)
The 'KeyError:...' equals the index of the IfcDoorPanelProperties...

**Comment by Moult** [comment]
IfcDoorPanelProperties are part of the old style properties in IFC2X3 that only exist for a few entities like doors and windows. It's currently only supported to view them, not to edit them. There's not a huge amount of motivation for supporting these as they are slowly getting removed anyway in favour of normal psets.

How urgent is this for you?

**Comment by lfcastel** [comment]
I wasn't aware these were getting removed. Based on what you said there I have no real motivation anymore to edit them so I'm okay if it is not getting implemented.

Thanks

**Comment by Moult** [comment]
We can keep it open because technically even in IFC4.3 it's still there, but it's just not a very nice place to put properties. Maybe one day someone or myself will be inspired enough to implement this :)

https://github.com/buildingSMART/IFC4.3.x-development/issues/89 is the upstream discussion about converting these types of "attribute entity psets" into normal psets.

**Comment by lfcastel** [comment]
Yes after your comment i went to the IFC 4.3 docs to check if it was labeled as deprecated, but indeed it's still there. Not to worry about the implementation, I'm just making an 'ideal' IFC residential house for educational purposes as an introduction to IFC. I think these properties are going to far anyway for this purpose...

Thanks
